libs/dal/src/repositories/topic/topic-subscribers.repository.ts

```diff
@@ -0,0 +1,16 @@
+import { AuthProviderEnum } from '@novu/shared';
+import { FilterQuery } from 'mongoose';
+
+import { TopicSubscribersEntity } from './topic-subscribers.entity';
+import { TopicSubscribers } from './topic-subscribers.schema';
+
+import { BaseRepository } from '../base-repository';
+
+export class TopicSubscribersRepository extends BaseRepository<
+  FilterQuery<TopicSubscribersEntity>,
+  TopicSubscribersEntity
+> {
+  constructor() {
+    super(TopicSubscribers, TopicSubscribersEntity);
+  }
+}
```

Comment:
As I have not yet 100% clear what methods to create to handle the addition and removal of subscribers from a topic, I will do the implementations while doing the needed use cases so I can also test it at the same time.

Reply:
Sound great to me.